### ISSUE TITLE
Count based on narrowPeak score

### ISSUE THREAD

**Issue by donaldcampbelljr** [databio/gtars#34]
Currently, we count overlaps in uniwig, but what if we could weigh the counts based on the associated score within a `narrowPeak` file?

**Comment by donaldcampbelljr** [comment]
@nleroy917 Per our discussion, it appears that this was not quite working as desired. Could you post some examples for later troubleshooting?

**Comment by nleroy917** [comment]
Yes, I've tried as best I could to isolate all this to a github repo here: https://github.com/nleroy917/uniwig-test/blob/master/README.md

**Comment by donaldcampbelljr** [comment]
Interestingly, I was curious why this conversion seems to be working for our recently included `bam` to `bw` workflow (which also uses bigtools behind the scenes). This workflow _streams_  bedGraph line by line to bigtools. Therefore, I  attempted to manually convert bedGraphs and used the `stdin` as input and used the uncompressed vs no compression, setting or not setting zoom flags and found:

![image](https://github.com/user-attachments/assets/d0f56767-562c-4335-892b-f94e0ab79c45)

Example command:
```
cat _core.bedGraph | ./target/release/bedgraphtobigwig -u - /hg38.chrom.sizes /bigtools_core_nozoom_uncompressed_stdin.bw 
```

So streaming bedGraph input seems to be working (and leaving the zoom to default).

There was also a previous discussion in this bigtools issue discussing zoomlevels and singlepass vs multipass: https://github.com/jackh726/bigtools/issues/63

This led me to also consider trying single pass mode while passing a bedGraph file (not using stdin), this also appears to provide a bw file in IGV that is expected, while adding a zoom level appears to cause issues, using default multipass mode with zoom=1 also works but all other zooms (including the default z=10) shows dicrepancy:
![image](https://github.com/user-attachments/assets/1805fa94-95f9-4cae-97f0-854f96e902b1)

### Recommendations
- option a) force zoom to 1 for these conversions (and continue to use multipass mode)
- option b) or, do single pass mode and remove zoom from the parameters.
- Future: we should be streaming these like we do for bam inputs (it will help with performance) but that will require future refactoring

**Comment by donaldcampbelljr** [comment]
This is solved and we will track streaming work in https://github.com/databio/gtars/issues/59
